controller/codebase.go

```diff
@@ -339,6 +379,11 @@ func (c *CodebaseController) Create(ctx *app.CreateCodebaseContext) error {
 		return jsonapi.JSONErrorResponse(ctx, err)
 	}
 
+	// new codebase is added register with analytics service
```

Comment:
We shouldn't register with the Analytics server on Workspace create. Codebase Create is in Space_Codebases

Reply:
done!